webview/src/setup/components/shared/Panels.tsx

```diff
@@ -0,0 +1,30 @@
+import {
+  VSCodePanelTab,
+  VSCodePanelView,
+  VSCodePanels
+} from '@vscode/webview-ui-toolkit/react'
+import React, { ReactNode } from 'react'
+
+interface PanelProps {
+  panels: { title: string; children: ReactNode }[]
+  className: string
+}
+
+export const Panels: React.FC<PanelProps> = ({ className, panels }) => {
```

Comment:
[F] The reason that I renamed this from `Slider` is because we already have a slider => webview/src/shared/components/slider/Slider.tsx